test/e2e/pages/quickInput.ts

```diff
@@ -19,17 +19,27 @@ export class QuickInput {
 	quickInputList: Locator;
 	quickInput: Locator;
 	quickInputTitleBar: Locator;
+	quickInputResult: Locator;
 
 	constructor(private code: Code) {
 		this.quickInputList = this.code.driver.page.locator(QUICK_INPUT_LIST);
 		this.quickInput = this.code.driver.page.locator(QuickInput.QUICK_INPUT_INPUT);
 		this.quickInputTitleBar = this.code.driver.page.locator(`.quick-input-title`);
+		this.quickInputResult = this.code.driver.page.locator(QuickInput.QUICK_INPUT_RESULT);
 	}
 
 	async expectTitleBarToHaveText(text: string): Promise<void> {
 		await expect(this.quickInputTitleBar).toHaveText(text);
 	}
 
+	async expectQuickInputResultsToContain(text: string[]): Promise<void> {
+		await test.step('Verify Quick Input results contain expected text', async () => {
+			for (let i = 0; i < text.length; i++) {
+				await expect(this.quickInputResult.filter({ hasText: text[i] })).toBeVisible();
+			}
+		});
+	}
+
```

Comment:
really small nit that you can ignore: it might be easier to parse this function if `text` was renamed to `titles`. 
```suggestion
	async expectQuickInputResultsToContain(titles: string[]): Promise<void> {
		await test.step('Verify Quick Input results contain expected title', async () => {
			for (let i = 0; i < titles.length; i++) {
				await expect(this.quickInputResult.filter({ hasText: titles[i] })).toBeVisible();
			}
		});
	}

```

Reply:
Totally. thanks!